app/core/src/screens/MMBPackageWrapper.js

```diff
@@ -39,26 +44,53 @@ export default class MMBPackageWrapper extends React.Component<
 
   fetchToken = async () => {
     const token = await AsyncStorage.getItem('mobile:MMB-Token');
-    this.setState({ token });
+    const simpleTokenData = await AsyncStorage.getItem(
+      'mobile:MMB-Simple-Token',
+    );
+
+    if (simpleTokenData) {
+      const { bookingId, simpleToken } = JSON.parse(simpleTokenData);
+      this.setState({ bookingId, simpleToken, token });
+    } else {
+      this.setState({ token, simpleToken: null, bookingId: null });
+    }
   };
 
   render = () => {
-    if (!this.state.token) {
+    if (
+      !this.state.token &&
+      (!this.state.bookingId && !this.state.simpleToken)
+    ) {
       return (
         <View style={styles.loginMessageContainer}>
           <Translation passThrough="You are not logged in, go to profile and log in" />
         </View>
       );
     }
 
-    return (
-      <ManageMyBookingPackage
-        onNavigationStateChange={this.props.onNavigationStateChange}
-        dimensions={this.props.dimensions}
-        currency="EUR"
-        accessToken={this.state.token}
-      />
-    );
+    if (this.state.bookingId && this.state.simpleToken) {
+      return (
+        <ManageMyBookingPackage
+          onNavigationStateChange={this.props.onNavigationStateChange}
+          dimensions={this.props.dimensions}
+          currency="EUR"
```

Comment:
Currency is hardcoded?

Reply:
This wrapper acts like the native application. It passes the props that the native package will pass to MMB.

It is not a part of the app in production.